src/Tasks.UnitTests/ResourceHandling/MSBuildResXReader_Tests.cs

```diff
@@ -37,6 +37,45 @@ public void ParsesSingleStringAsString()
             AssertSingleStringResource(resxWithSingleString, "StringResource", "StringValue");
         }
 
+        [Fact]
+        public void ParsesSingleStringWithoutPreserveAsString()
+        {
+            var resxWithSingleString = MSBuildResXReader.GetResourcesFromString(
+                ResXHelper.SurroundWithBoilerplate(
+                    @"<data name=""StringResource"">
+    <value> StringValue </value>
+    <comment>Comment</comment>
+  </data>"));
+
+            AssertSingleStringResource(resxWithSingleString, "StringResource", " StringValue ");
```

Comment:
Is this what the .NET Framework path does? I would have expected it to be `"StringValue"` with no leading/trailing space.

Reply:
Yes. I deliberately test that and was surprised as you are. It's weird, but seems to be that how it always was.

Reply:
Well, I am indeed surprised! Thanks for double-checking.